src/app/options/qgsoptions.cpp

```diff
@@ -1292,8 +1292,11 @@ void QgsOptions::checkPageWidgetNameMap()
   };
   traverseModel( QModelIndex() );
 
-  Q_ASSERT_X( pageNames.count() == pageTitles.count(),  "QgsOptions::checkPageWidgetNameMap()", QStringLiteral( "QgisApp::optionsPagesMap() is outdated, contains too many entries, "
-              " this is often a problem with missing translations for the entries (extra entries: %1)" ).arg( qgsSetJoin( QSet( pageNames.keyBegin(), pageNames.keyEnd() ) - pageTitles.toSet(), QStringLiteral( "," ) ) ).toLocal8Bit().constData() );
+  Q_ASSERT_X( pageNames.count() == pageTitles.count(),  "QgsOptions::checkPageWidgetNameMap()",
+              QStringLiteral( "QgisApp::optionsPagesMap() is outdated, contains too many entries, "
+                              " this is often a problem with missing translations for the entries (extra entries: %1)" ).arg(
+                qgsSetJoin( QSet( pageNames.keyBegin(), pageNames.keyEnd() ) - QSet( pageTitles.constBegin(), pageTitles.constEnd() ),
+                            QStringLiteral( "," ) ) ).toLocal8Bit().constData() );
```

Comment:
Wanted changes ?

Reply:
Yes, I need to remove the call to toSet() for pageTitles which doesn't exist anymore in Qt6